department_names.py

```diff
@@ -4,61 +4,86 @@
 import logging
 import os
 import requests
-import yaml
+import csv
 
-portal_url = os.environ.get('PORTAL_URL', "https://vulekamali.gov.za/")
+portal_url = os.environ.get("PORTAL_URL", "https://vulekamali.gov.za/")
 
 department_names = {
-    'national': {},
-    'provincial': {},
+    "2014-15": {"national": {}, "provincial": {},},
```

Comment:
rather than hardcoding years, it should check if a year exists, and insert it into this dictionary if it doesn't exist.

Reply:
get_financial_year() function created.

https://github.com/vulekamali/treasury-pipelines/pull/33/commits/4b1a49e1eb9b682d40d121d98f62397c6599a374